rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_limits_urdf.py

```diff
@@ -33,12 +33,15 @@ def get_joint_limits(key='robot_description', use_smallest_joint_limits=True):
             if jtype == 'fixed':
                 continue
             name = child.getAttribute('name')
+            try:
+                limit = child.getElementsByTagName('limit')[0]
+            except:
+                continue
```

Comment:
You know I had that exact same though, and actually [pushed that change](https://github.com/ros-controls/ros_controllers/commit/3e5b9c034c5bc7b46ab411086288ac9a521cf8b1) before this PR. But it turns out that because we parse the entire URDF (we don't know which joints will end up being controlled by a JTC), this potentially generates lots of spam, from joints that might never be the business of this plugin.

I'm open to better suggestions, but my initial, naïve solution didn't work out well.

I'll see if I can find a non-spamming way of alerting about controllers that are being left out because of missing joint limits.